added volume and time metric charts to dashboard

hcaballero2 opened the pull request:
Added charts to Team metrics tab: Time based metrics and Volume Based metrics. These charts will allow users to view both their overall team and independent metrics for these things. Time based metrics is based on days so how many days it took to complete a pr, issue, commit, etc. And volume is based on count so how many PRs, issues, commits, etc were completed.

viswanathreddy1017 commented:
Nice work Hazel, I believe these are the static progress bars as I kept changing the dropdown values the data is same

One thing I noticed is the manual metrics Header Bar is not as expected 
<img width="1659" height="884" alt="Screenshot 2025-10-27 214739" src="https://github.com/user-attachments/assets/b700b0a0-34b4-4ab3-9843-c88ef654a18d" />
<img width="1665" height="812" alt="image" src="https://github.com/user-attachments/assets/0bcfbb9e-e323-4532-ae0f-5a7aaea9c1a9" />
and the names are overlapping

hcaballero2 commented:
> filter by user in metrics tab
> fixed rendering issue in team tab
> changed metrics tab chart style and added user
and sprint filtering
^ updated script so request PR files are now different. i fixed the issue you mentioned above in this new code!
